### ISSUE TITLE
App crashing on IOS (Prod) blinking/flickering in development

### ISSUE THREAD

**Issue by hraschan** [computerjazz/react-native-draggable-flatlist#449]
*This package has external dependencies of [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated) and [react-native-gesture-handler](https://github.com/software-mansion/react-native-gesture-handler) which must be installed separately. Before opening an issue related to animations or gestures please verify that you have completed ALL installation steps, including the [changes to MainActivity.](https://software-mansion.github.io/react-native-gesture-handler/docs/getting-started.html#android)*

**Describe the bug**
When using Draggable Flatlist in production the app first is freezing when I try to drag an Item. After a minute the App crashes. When I'm using the list in development I noticed a flicker/blinking of the list every time I drag an item to the new position.

`  const onDragEnd = useCallback(({ data }: DragEndParams<WorkoutExercise>) => {
    dispatch(
      workoutActions.updateWorkoutExercises(
        [...data].map((item, index) => {
          return { ...item, sort: (index + 1).toString() };
        })
      )
    );
  }, []);` 

```
 <DraggableFlatList
      contentContainerStyle={{
        paddingBottom: 160,
      }}
      keyboardShouldPersistTaps="handled"
      // style={{ flex: 1 }}
      showsVerticalScrollIndicator={false}
      data={workoutExercises}
      onDragEnd={onDragEnd}
      renderItem={renderItem}
      keyExtractor={(item) => item.id}
    />
```

**To Reproduce**
Link a snack if possible.

**Platform & Dependencies**
Please list any  applicable dependencies in addition to those below (react-navigation etc).
 - react-native-draggable-flatlist version: 4.0.0
 - Platform: IOS 
 - React Native or Expo version: 47.0.8
 - Reanimated version: 2.12.0
 - React Native Gesture Handler version: 2.8.0

**Additional context**

**Comment by ps-dima** [comment]
@hraschan , were you able to solve this issue ? I'm currently facing the same bug :(

**Comment by tara-singh-danu** [comment]
Facing the same issue

**Comment by saurabhg138** [comment]
Yes I also notice the flicker/blinking on android as well

**Comment by hraschan** [comment]
Nope. I didn’t find any solution. This issue doesn’t come up every time 🥲

**Comment by neosh11** [comment]
Yup same issue, it's constantly flashing
